cmd/minikube/cmd/start_flags.go

```diff
@@ -108,6 +108,7 @@ const (
 	kicBaseImage            = "base-image"
 	startOutput             = "output"
 	ports                   = "ports"
+	dockerNetwork           = "docker-network"
```

Comment:
this can be generalized for other drivers such as Podman
there is no need for it to be called docker-network

Reply:
would you prefer `--container-network`? just `--network` seems too general to me

Reply:
I think --network is good enough, other drivers actually can implement this feature, we can later add same option for KVM and others.
but till then we should Exit with usage and say this feature is only currently suported in docker driver.

Reply:
done